### ISSUE TITLE
power level changes can lie about who is changing whose power. eg othervdh opped richvdh in a room but it claimed richvdh did instead

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#7751]

**Comment by ara4n** [comment]
or vice versa actually

**Comment by richvdh** [comment]
It's certainly not a lie, just unclear.

It was shown as below. This was richvdh opping othervdh.

![image](https://user-images.githubusercontent.com/1389908/49040880-a9845300-f1bb-11e8-931c-3ebc85889887.png)

**Comment by lampholder** [comment]
Right, so there's a UX problem, and critically the information you'd need to _unconfuse_ yourself is only accessible by digging into the event.

I think we just need to update the UX for these events, either always putting the full disambiguated `Display Name (mxid)` right into the rendering or have it available on hover.
